BrainPortal/app/views/resource_usage/_resource_usage_table.html.erb

```diff
@@ -0,0 +1,297 @@
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+-%>
+
+<% title @maintype =~ /Userfile/    ? 'File Deltas'      :
+         @maintype =~ /Cputime/     ? 'Task CPU times'   :
+         @maintype =~ /Walltime/    ? 'Task Walltimes'   :
+         @maintype =~ /Space.*Task/ ? 'Task Final Space' : "" %>
+
+<div class="menu_bar">
+  <% buttons =
+     [ [ 'Userfile Disk Space', 'SpaceResourceUsageForUserfile'      ],
+       [ 'Task CPU Time',       'CputimeResourceUsageForCbrainTask'  ],
+       [ 'Task Walltime',       'WalltimeResourceUsageForCbrainTask' ],
+       [ 'Task Final Status',   'SpaceResourceUsageForCbrainTask'    ],
+     ]
+  %>
+
+  <% buttons.each do |pair|
+       button_label, maintype = pair
+  %>
+    <%= link_to button_label,
+        url_for(scope_filter_params(@scope, :set, {
+          :attribute => 'type',
+          :value     => maintype,
+        })),
+        :class => "button menu_button"
+    %>
+  <% end %>
+</div>
+
+<p>
+
+<fieldset>
+  <legend>Usage Summary</legend>
+  <p class="medium_paragraphs">
+  This is a <em>very wide</em> report table. It contains
+  resource usage records for disk space and time consumed.
+  Feel free to hide columns using the <strong>+</strong>
+  menu at the left side of the main table header.
+  <p class="medium_paragraphs">
+  <% if    @maintype == 'SpaceResourceUsageForUserfile'      %>
+    It displays the <span color="green">changes of the sizes</span> of all the files,
+    existing as well as deleted.
+  <% elsif @maintype == 'CputimeResourceUsageForCbrainTask'  %>
+    It displays the <span color="green">CPU time</span> accumulated by tasks.
+  <% elsif @maintype == 'WalltimeResourceUsageForCbrainTask' %>
+    It displays the <span color="green">'wall' time</span> accumulated by tasks.
+  <% elsif @maintype == 'SpaceResourceUsageForCbrainTask'    %>
+    It displays the <span color="green">disk space</span> in the work directory,
+    as well as the <span color="green">final status</span> of past tasks. The disk space
+    is not garanteed to be accurate, as this is a expensive resource to compute
+    and is only provided FYI. The main purpose of this report is to compare the
+    final <strong>status</strong>.
+  <% end %>
+  <p class="medium_paragraphs">
+  Columns labeled <strong>Cached</strong> record pieces of information
+  about resources <em>as they were at the time the record was made</em>.
+  If a resource was destroyed (e.g. a non-<strong>Cached</strong> column is empty), these stay behind
+  and provide filtering options. It makes it possible to compute resource usage reports for
+  deleted resources.
```

Comment:
![report](https://user-images.githubusercontent.com/777590/65173328-8eb9ee00-da1c-11e9-8e1c-806ae68e9d10.png)

I played with the interface. It is not clear that the report takes into account the filter applied to the table.

Reply:
Can you be more specific? Please give an exact example?

Reply:
By design, whatever filer you apply sticks around between reports. That's so you can say for example "I want to see user=abc and task_type=xyz" and switch between reports with these two filters applied.

Reply:
![report1](https://user-images.githubusercontent.com/777590/65174598-36382000-da1f-11e9-99f3-224d4950fdac.png)

If I have no filter apply to my table I get: 
> 6 usage records: + 25.6 Kb - 319.5 Mb (Total: - 319.5 Mb) (Average: - 53.2 Mb) 
And if I have `Remote resource` NaN selected I get: 
> 1 usage record: + 0 bytes - 8.5 Kb

For me, it just not clear that the report applies the table filter.

Reply:
I don't get it but we'll talk about this when we meet at the office...

Reply:
Got the explanation in person, will work on it now.